settings.php

```diff
@@ -934,6 +947,21 @@ function convertseconds($argument)
                                                     <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
                                                 </div>
                                                 <br>
+                                                <h4><a id="ratelimit"></a>Rate-limiting</h4>
+                                                <p>Block clients making more than <input type="number" name="rate_limit_count" value="<?=$rate_limit_count?>"> queries within
```

Comment:
```suggestion
                                                <p>Block clients making more than <input type="number" name="rate_limit_count" value="<?=$rate_limit_count?>" min="0" step="10"> queries within
```

Reply:
If you want to limit the width of the field, add `size="6"` to the input tag.

Reply:
`size="6"` doesn't change anything.

Reply:
Yeah... 
Apparently only Firefox respects `size` attribute. `size="6"` is not respected (by most browsers) if the type is `number`.

If you want to use `type="number"` we need to add a CSS rule.

Reply:
I don't think we can/should set a maximum. One might argue that a sensible maximum could be something like 100,000 queries over 86,400 seconds (one day) but users may want to do other things (like limiting per week) - you never know. I'd feel better without enforcing a maximum here.

Reply:
Addressed in the most recent commit, preview:

![Screenshot from 2022-02-26 10-08-04](https://user-images.githubusercontent.com/16748619/155837382-dcc829a9-0c67-4a9b-8139-96aa8447569d.png)

The query limit field is intentionally larger than the seconds field as we are expecting larger numbers here.